### ISSUE TITLE
Show contact images in recent, call screen, contact interfaces

### ISSUE THREAD

**Issue by pranav-vaid** [Chooloo/koler#466]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

It would be good to have contact images in the app.

#### How will you/everyone benefit from this feature?

It will make the UI/UX more intuitive.

**Comment by roeiedri** [comment]
Contact images are already shown.
When theres no contact imags, nothing will be shown in order to save more space on screen.
Did you mean to have a placeholder when theres no contact image?

**Comment by DodoLeDev** [comment]
I've a similar problem with recents contacts profile pictures!

On the recents screen, there is no contact image, while on the contacts page, no problem to show them!

**Comment by pranav-vaid** [comment]
Yes actually. @DodoLeDev is right. Recent screen doesnot show contact image.

On a side note, please reduce the user interaction while calling someone. I have to click on screen 3 times just to make a call from recents screen. That is definitely not a good UX. 

Or if i am missing something, please tell me.
